### PR TITLE
chore: standardize prefix language of questions

### LINKED ISSUE #73: Standardize prefix of questions with language context
Great work with this project and making it really accessible to quickly test your programming knowledge on the fly.

One thing I noticed is given that now there are multiple languages being quizzed, it can sometimes be ambiguous as to what language is being tested on.

For example, I was given a question on what the method gives Linux time and the answer was getTime(). Although the word "method" suggests it is, correctly, JavaScript.

If a question is broad or specific enough per subject, then this change isn't necessary for those questions. One example question that probably doesn't need a change is:

> Which of the following techniques is implemented to improve CPU performance?

## My proposal examples:

**Before**

> Which of the following methods returns the month in the specified date according to local time, as a zero-based value?

**After**

> In JavaScript, which of the following methods returns the month in the specified date according to local time, as a zero-based value?

**Before**

> Which of the following is a wildcard character that can be used with the SQL LIKE statement?

**After**

> In SQL, which of the following is a wildcard character that can be used with the LIKE statement?

### PR DESCRIPTION
**Pull request by erictleung**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #73

### CODE REVIEW

File: src/data/full-quiz.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ const fullQuiz = [
   },
   {
     Question:
-      "In a switch statement, the __________________ case executes if none of the options are true.",
+      "In JavaScript, in a switch statement, the __________________ case executes if none of the options are true.",
```
Comment (jdwilkin4) [inline src/data/full-quiz.ts:91]: I think the wording can be clean up a bit

Proposed fix:
In a JavaScript switch statement, the __________________ case executes if none of the options are true.

File: src/data/full-quiz.ts [hunk 2]
```diff
@@ -4854,7 +4851,7 @@ const fullQuiz = [
     Link: "https://www.freecodecamp.org/news/the-linux-commands-handbook/#the-linux-wc-command",
   },
   {
-    Question: "Which one is NOT used with justify-content?",
+    Question: "In Flexbox, which one is NOT used with justify-content?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.ts:4854]: I would change it to CSS Flexbox

File: src/data/full-quiz.ts [hunk 3]
```diff
@@ -5552,7 +5549,7 @@ const fullQuiz = [
     Link: "https://devdocs.io/css/color_value/rgba()",
   },
   {
-    Question: "Which one is NOT used with align-items?",
+    Question: "In Flexbox, which one is NOT used with align-items?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.ts:5552]: I would specify CSS Flexbox

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
I think it looks good.
Just left a couple small comments 👍

**Comment by erictleung** [comment]
Thanks for the feedback! I've found some more "flexbox" references that could be clarified as well and changed those too. Take a look when you have a chance.

**Pull request merged**